kale/evaluate/metrics.py

```diff
@@ -0,0 +1,22 @@
+def c_index(y, y_pred):
+    """
+    Calculate Concordance Index (CI), which is a metric to measure proportion of concordant pairs between real and
```

Comment:
- Insert "the" before "Concordance Index", before "proportion".
- Provide a hyperlink to concordant pairs (I do not know what it means), e.g. wiki is usually the first choice (if available) https://en.wikipedia.org/wiki/Concordant_pair

Reply:
Resolved and add kale.evaluate in documentation.